examples/llm/benchmarking.md

```diff
@@ -0,0 +1,333 @@
+    --synthetic-input-tokens-mean ${isl} \
+    --synthetic-input-tokens-stddev 0 \
+    --output-tokens-mean ${osl} \
+    --output-tokens-stddev 0 \
+    --extra-inputs max_tokens:${osl} \
+    --extra-inputs min_tokens:${osl} \
+    --extra-inputs ignore_eos:true \
+    --concurrency ${concurrency} \
+    --request-count $(($concurrency*10)) \
+    --warmup-request-count $(($concurrency*2)) \
+    --num-dataset-entries $(($concurrency*12)) \
+    --random-seed 100 \
+    -- \
+    -v \
+    --max-threads 256 \
+    -H 'Authorization: Bearer NOT USED' \
+    -H 'Accept: text/event-stream'
+
+done
+```
+Key settings:
+* **streaming**: Enables token streaming for more realistic performance measurements.
+* **concurrency**: Number of simultaneous requests.
+* **isl**: Input sequence length.
+* **osl**: Output sequence length requested.
+
+## Disaggregated Multi Node Benchmarking
+
+Two H100 80GB x8 node is required for this setup.
+
+With the Dynamo repository and the benchmarking image available, perform the following steps:
+
+1\. Run benchmarking container (node 0 & 1)
+```bash
+./container/run.sh -it
+```
+Note: Make sure to run it in the `dynamo` directory cloned.
+
+2\. Download model (node 0 & 1)
+```bash
+huggingface-cli download neuralmagic/DeepSeek-R1-Distill-Llama-70B-FP8-dynamic
+```
+
+3\. Start NATS and ETCD (node 0)
+```bash
+nats-server -js -p 4222 -m 8222 &
+etcd --listen-client-urls http://0.0.0.0:2379 --advertise-client-urls http://0.0.0.0:2379 &
+```
+
+4\. Config NATS and ETCD (node 1)
+```bash
+export NATS_SERVER="nats://<node_0_ip_addr>"
+export ETCD_ENDPOINTS="<node_0_ip_addr>:2379"
+```
+Note: Node 1 must be able to reach Node 0 over the network for the above services.
+
+5\. Update `/workspace/examples/llm/configs/disagg_router.yaml` to the following (node 0 & 1)
+```yml
+Frontend:
+  served_model_name: neuralmagic/DeepSeek-R1-Distill-Llama-70B-FP8-dynamic
+  endpoint: dynamo.Processor.chat/completions
+  port: 8000
+
+Processor:
+  model: neuralmagic/DeepSeek-R1-Distill-Llama-70B-FP8-dynamic
+  block-size: 128
+  max-model-len: 16384
+  router: kv
+
+Router:
+  model-name: neuralmagic/DeepSeek-R1-Distill-Llama-70B-FP8-dynamic
+  min-workers: 1
+
+VllmWorker:
+  model: neuralmagic/DeepSeek-R1-Distill-Llama-70B-FP8-dynamic
+  kv-transfer-config: '{"kv_connector":"DynamoNixlConnector"}'
+  block-size: 128
+  max-model-len: 3500
+  remote-prefill: true
+  tensor-parallel-size: 4
+  gpu-memory-utilization: 0.95
+  disable-log-requests: true
+  router: kv
+  enable-prefix-caching: true
+  ServiceArgs:
+    workers: 2
+    resources:
+      gpu: 4
+
+PrefillWorker:
+  model: neuralmagic/DeepSeek-R1-Distill-Llama-70B-FP8-dynamic
+  kv-transfer-config: '{"kv_connector":"DynamoNixlConnector"}'
+  block-size: 128
+  max-model-len: 3500
+  max-num-batched-tokens: 3500
+  tensor-parallel-size: 1
+  gpu-memory-utilization: 0.95
+  disable-log-requests: true
+  ServiceArgs:
+    workers: 8
+    resources:
+      gpu: 1
+```
+
```

Comment:
we should add "key settings" as above and describe the config here - any deltas with single node can be added as key settings to emphasize the delta

Reply:
Will address multi-node config in a follow up PR.